biorxiv_experiment/03_run_taggers_on_preprints.ipynb

```diff
@@ -0,0 +1,246 @@
+{
```

Comment:
<p>Ooh fun, I see you've added Java and Perl dependencies because Python package management is too easy :laughing:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/biovectors/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>